charts/connectors/testdata/template-cases.golden.txtar

```diff
@@ -10675,8 +10637,7 @@ spec:
             weight: 100
       automountServiceAccountToken: false
       containers:
-      - command: null
-        env:
+      - env:
```

Comment:
👍 on the fix @RafalKorepta

@chrisseto here's a part of the diff to which Rafal is referring to.

Reply:
Right but when the K8s API JSON unmarshals (You've read sig YAML, it's just JSON)`command: null` into a `Container` struct, there is no distinction between `Command` not being specified and being explicit set to `null`/`nil`. The behavior is present for many other fields that show up as `null` (e.g. `creationTimestamp`).


https://github.com/user-attachments/assets/ebd8c215-3376-441c-8a74-3d4c92ebe318